docs/source/intro/installation.rst

```diff
@@ -1,16 +1,892 @@
+
+      .. code-block:: console
+
+         $ sudo port install postgresql postgresql-server rabbitmq-server
+         $ pg_ctl -D /usr/local/var/postgres start
+         $ curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
+         $ python get-pip.py
+         $ pip install aiida-core
+
+      See :ref:`MacOS X (MacPorts) instructions<intro:install:prerequisites:macports>` for details.
+
+   .. dropdown:: Gentoo Linux
+
+      .. code-block:: console
+
+         $ emerge -av git postgresql rabbitmq-server
+         $ rc-update add rabbitmq
+         $ emerge --ask dev-lang/python:3.7
+         $ emerge --ask dev-python/pip
+         $ pip install aiida-core
+
+      See :ref:`Gentoo Linux instructions<intro:install:prerequisites:gentoo>` for details.
+
+   .. dropdown:: Windows Subsystem for Linux
+
+      .. code-block:: console
+
+         $ sudo apt-get install \
+            postgresql postgresql-server-dev-all postgresql-client \
+            git python3-dev python-pip
+         $ sudo service postgresql start
+         # install rabbitmq on windows
+         $ pip install aiida-core
+
+      See :ref:`WSL instructions<intro:install:prerequisites:wsl>` for details.
+
+.. _intro:install:prerequisites:ubuntu:
+
+Ubuntu
+------
+
+To install the prerequisites on Ubuntu and any other Debian derived distribution, you can use the ``apt`` package manager.
+The following will install the basic ``python`` requirements and the ``git`` source control manager:
+
+.. code-block:: console
+
+   $ sudo apt-get install git python3-dev python3-pip
+
+To install the requirements for the ``postgres`` database run the following:
+
+.. code-block:: console
+
+   $ sudo apt-get install postgresql postgresql-server-dev-all postgresql-client
+
+Install the RabbitMQ message broker:
+
+.. code-block:: console
+
+   $ sudo apt-get install rabbitmq-server
+
+This installs and adds RabbitMQ as a system service. To check whether it is running:
+
+.. code-block:: console
+
+   $ sudo rabbitmqctl status
+
+If it is not running already, it should after a reboot.
+
+Finally install the aiida-core python environment:
+
+.. code-block:: console
+
+   $ sudo git python3-dev python3-pip
+   $ pip install aiida-core
+
+.. admonition:: Further Reading
+   :class: seealso title-icon-read-more
+
+   - For a more detailed description of database requirements and usage see the :ref:`database topic<topics:database>`.
+   - For problems with installing RabbitMQ, refer to the detailed instructions provided on the `RabbitMQ website for Debian based distributions <https://www.rabbitmq.com/install-debian.html>`_.
+
+.. _intro:install:prerequisites:brew:
+
+Mac OS X (homebrew)
+-------------------
+
+For Mac OS we recommend using the `Homebrew`_ package manager.
+If you have not installed Homebrew yet, you can do so with the following command:
+
+.. code-block:: console
+
+   $ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
```

Comment:
The recommended way to install homebrew could change in the future, maybe we just link to install instructions instead?

Reply:
Fixed in 0cf7a7d